pkg/gofr/datasource/opentsdb/preprocess.go

```diff
@@ -11,15 +11,9 @@ import (
 	"strings"
 )
 
-// defaultTransport defines the default HTTP transport settings,
-// including connection timeouts and idle connections.
-var defaultTransport = &http.Transport{
-	MaxIdleConnsPerHost: 10,
-	DialContext: (&net.Dialer{
-		Timeout:   defaultDialTime,
-		KeepAlive: connectionTimeout,
-	}).DialContext,
-}
+var (
+	ErrInvalidDataPoint = errors.New("invalid data points")
```

Comment:
Should we keep this exported also?

And we don't need to use () for single var declaration.